docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
 
-* Location in archive: conditional/namespaces/{namespace}/pods/{pod}/{pod}.json
-* Id in config: conditional/pod_definition
-* Since versions:
-  - 4.11+
+### Location in archive
+- `conditional/namespaces/{namespace}/pods/{name}/{name}.json`
+
+### Config ID
+`conditional/pod_definition`
+
+### Released version
+- 4.11.0
+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## PodDisruptionBudgets
 
-gathers the cluster's PodDisruptionBudgets.
+Collects the cluster's `PodDisruptionBudgets`.
+
+### API Reference
+- https://github.com/kubernetes/client-go/blob/v11.0.0/kubernetes/typed/policy/v1beta1/poddisruptionbudget.go#L80
+- https://docs.okd.io/latest/rest_api/policy_apis/poddisruptionbudget-policy-v1beta1.html
 
-The Kubernetes api https://github.com/kubernetes/client-go/blob/v11.0.0/kubernetes/typed/policy/v1beta1/poddisruptionbudget.go#L80
-Response see https://docs.okd.io/latest/rest_api/policy_apis/poddisruptionbudget-policy-v1beta1.html
+### Sample data
+- [docs/insights-archive-sample/config/pdbs/openshift-machine-config-operator/etcd-quorum-guard.json](./insights-archive-sample/config/pdbs/openshift-machine-config-operator/etcd-quorum-guard.json)
+
+### Location in archive
+- `config/pdbs/{namespace}/{name}.json`
+
+### Config ID
+`clusterconfig/pdbs`
 
-* Location in archive: config/pdbs/
-* See: docs/insights-archive-sample/config/pdbs
-* Id in config: clusterconfig/pdbs
-* Since versions:
-  - 4.4.30+
-  - 4.5.34+
-  - 4.6+
+### Released version
+- 4.6.0
+
+### Backported versions
+- 4.4.30+
+- 4.5.15+
+
+### Changes
+None
 
 
 ## SAPConfig
 
-collects selected security context constraints
+Collects selected security context constraints
 and cluster role bindings from clusters running a SAP payload.
 
-**Conditional data**: This data is collected only if the "installers.datahub.sap.com" resource is found in the cluster.
+### API Reference
+- https://pkg.go.dev/github.com/openshift/client-go/authorization/clientset/versioned/typed/authorization/v1
+- https://pkg.go.dev/github.com/openshift/client-go/security/clientset/versioned/typed/security/v1
 
-Relevant OpenShift API docs:
-  - https://pkg.go.dev/github.com/openshift/client-go/authorization/clientset/versioned/typed/authorization/v1
-  - https://pkg.go.dev/github.com/openshift/client-go/security/clientset/versioned/typed/security/v1
+### Sample data
+- [docs/insights-archive-sample/config/securitycontextconstraint](./insights-archive-sample/config/securitycontextconstraint)
+- [docs/insights-archive-sample/config/clusterrolebinding](./insights-archive-sample/config/clusterrolebinding)
 
-* Location in archive: config/securitycontentconstraint/, config/clusterrolebinding/
-* Id in config: clusterconfig/sap_config
-* Since versions:
-  - 4.6.20+
-  - 4.7+
+### Location in archive
+- `config/{securitycontentconstraint|clusterrolebinding}/{name}.json`
```

Comment:
Could we avoid unnecessary complexity in path templates. It is hard enough to figure out what the paths are if there have been any changes.

```suggestion
- `config/clusterrolebinding/{name}.json`
- `config/securitycontentconstraint/{name}.json`
```